app/partials/data-export-button.cjsx

```diff
@@ -16,13 +19,19 @@ module.exports = React.createClass
 
   requestDataExport: ->
     @setState exportError: null
-    apiClient.post @props.project._getURL(@props.exportType), media: content_type: 'text/csv'
+    apiClient.post @props.project._getURL(@props.exportType), media: content_type: @props.contentType
       .then =>
         @_exportsGet = null
         @setState exportRequested: true
       .catch (error) =>
         @setState exportError: error
 
+  recentAndReady: (exported) ->
+    exported? and (exported.metadata.state is 'ready' or not exported.metadata.state?)
```

Comment:
can this read, `exported? and (exported.metadata.state in [ 'ready', 'finished' ] or not exported.metadata.state?)` API has the finished state currently.